src/JsonpCallbackValidator.php

```diff
@@ -64,6 +64,10 @@ class JsonpCallbackValidator
      */
     public static function validate($callback)
     {
+        if (!is_string($callback)) {
```

Comment:
the argument is documented as `@param  string  $callback`

Reply:
Very true, but as a validator, the more we validate, the better.

In fact, FOSRoutingBundle will call validate() with an array for URLs like `...?callback=[]`. We could fix this in FOSRoutingBundle and other callers, or fix it here for everyone.

Reply:
sounds good